code-reviews.md

```diff
@@ -0,0 +1,74 @@
+
+We recommend considering a fine granularity of code reviews.
+The benefit is both temporal and qualitative.
+By doing code reviews as frequently as possible, the exercise is made easier and all members feel more involved in development.
+By doing light and targeted code reviews on a specific topic, the review is done with more attention because it is less complex.
+
+A code review is an opportunity to **actively** communicate about the changes to other developers and get feedback on the quality of the changes.
+As far as possible, a code review should be done for small changes.
+In this perspective, we advise against doing code reviews for a collection of changes that can be analyzed independently.
+However, it is important to keep in mind that what is under review must be understandable in its software context.
+Complex refactorings or features can not be splitted in small changes as they would lose software design information.
+The review would then bypass the algorithmic choices in favor of a simple technical review.
+
+When starting a review, we recommend to assess the granularity of the review to keep visible the algorithm and design of the feature.
+
+## Best practices for reviewing codes
+
+The challenge of a code review is to position oneself in the face of another member's work production.
+It is not a question of sanctioning his/her work, but of assessing it; issuing an opinion.
+
+Here are 5 good practices to adopt that may be beneficial for a code review.
+
+1. Engage conversation for clarification 💬
+
+The point here is to question what has been produced. It is not because the proofreader has difficulty understanding the code that the quality of the code is poor.
+Question the developer by asking him why he did it rather than being prescriptive.
+Code reviews is an exercise that enhances the value of the work produced in a team spirit.
+
+2. Have a pragmatic state of mind ✔️
+
+It is important to keep in mind that what is provided is functional.
+External parameters can force the developer not to provide optimal code.
+He/she is not omniscient and learns from teamwork too.
+Having a functional code is the paramount issue (considering that good practices of coding are respected).
+
+3. Review as often as possible 🕙
+
+Code reviews will be relevant and beneficial if the exercise is frequent within the team.
+It is not the business of a few members but of everyone because they contribute to the good integration and involvement of everyone.
+
+4. Focus on the main 🧠
+
+A code review should not be overthought.
+The reviewer should focus on the algorithm and the code.
+With a problem comes a solution.
+It must be possible to be synthetic and have a global vision of what is in question.
+
+5. Communicate positively and with consideration 😃
+
+Finally, the most important is the way to communicate.
+One has to keep a warm and team spirit.

```

Comment:
I find it weirdly said

```suggestion
We want to keep a warm and team spirit
```

Reply:
You're right. It's better to keep talking about OUR point of view, as I did the whole document long.